### DISCUSSION TITLE
[#1040945] Axis alignment widget in the top right of the screen

### DISCUSSION THREAD

**Discussion by dacowan** [Mojang/minecraft-editor#68] [Feedback]
I would like a colored axis alignment widget in the top right of the screen that shows the +ve direction in which the X, Y and Z axis’ are pointing.  The widget should rotate as the player rotates, showing the axis’ directions relative to the player view.
In addition, the color of the axis’ on the widget should correspond with the Vector Text Field labels for coordinates in the UI input components.

**Comment by minecraftchloe** [comment]
I would also like this! ;)
